micrometer-tracing-tests/micrometer-tracing-integration-test/src/main/java/io/micrometer/tracing/test/SampleTestRunner.java

```diff
@@ -454,9 +454,12 @@ private static void checkTracingSetupAssumptions(TracingSetup tracingSetup, Trac
         }
 
         private static void checkZipkinAssumptions(Sender sender) {
-            CheckResult checkResult = sender.check();
-            Assumptions.assumeTrue(checkResult.ok(),
-                    "There was a problem with connecting to Zipkin. Will NOT run any tests");
+            try {
+                sender.sendSpans(Collections.emptyList()).execute();
```

Comment:
this makes migration easier as the new simpler senders have no check function anymore.

See https://github.com/openzipkin/zipkin-reporter-java/releases/tag/3.2.1 for an example

Reply:
when this project updates to reporter 3.2+, this line will drift slightly due to no more Call type. Notably, no more `execute()` and the function is `send` not `sendSpans`

```java
sender.send(Collections.emptyList());
```